### PR TITLE
add html css and image files

### PR DESCRIPTION
**Pull request by EyasuTeshome**
add html css and image files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    Entry</a>
+            </h3>
+
+
+        </div>
+
+
+
+
+    </div>
```
Comment (EyasuTeshome) [inline index.html:99]: Please kindly do not add blank lines, spaces, or indentations without a reason. They will make your code look less professional.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div id="main">
```
Comment (EyasuTeshome) [inline index.html:13]: - please consider changing the `div `element to `header` for HTML semantic reasons

File: README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
```
Comment (EyasuTeshome) [inline README.md:1]: It's good that you included readme file, but kindly make sure that your readme file properly explains what you did on the project. considering this I will highly encourage you to re-write the readme file.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @EyasuTeshome,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] Kindly edit the PR's title to something more descriptive. it can be "Review of Past Project PR or Review Trial Project".
- [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**